resources/rebuild.sh

```diff
@@ -134,80 +134,74 @@ EOF
     rm -rf $INITRAMFS_BUILD
 }
 
-
-function get_linux_git {
-    # git clone -s -b v$KV ../../linux
-    # --depth 1
-    cd linux
-    LATEST_TAG=$(git tag -l "v$KV.*" --sort=v:refname |tail -1)
-    git clean -fdx
-    git checkout $LATEST_TAG
+function clone_amazon_linux_repo {
+  [ -d linux ] || git clone https://github.com/amazonlinux/linux linux
```

Comment:
it seems indentation for this file was 4 spaces. could we keep consistency?

Reply:
<irrelevant-to-PR>that's why we should be using TABs at the beginning of lines</irrelevant-to-PR>

Reply:
My IDE has seemingly decided that 1 tab = 2 space in .sh files. Anyway, fixed